src/Core_Command.php

```diff
@@ -129,8 +129,7 @@ public function download( $args, $assoc_args ) {
 			}
 
 			WP_CLI::log( sprintf( "Creating directory '%s'.", $download_dir ) );
-			$mkdir = \WP_CLI\Utils\is_windows() ? 'mkdir %s' : 'mkdir -p %s';
-			WP_CLI::launch( Utils\esc_cmd( $mkdir, $download_dir ) );
+			mkdir( $download_dir, 0777, true /*recursive*/ );
```

Comment:
Ta, fixed. Re `WP_CLI_STRICT_ARGS_MODE=1`, without it `ABSPATH` is set to the value of `--path` and `$assoc_args['path']` is unset - `$assoc_args['path']` is only set if strict mode is. This has plagued me a bit on other commands, as behaviour changes in confusing ways. Perhaps introducing alternative names for options that clash with global ones (eg `--install-path`) would be useful (probably been suggested before...).